src/generic_ui/polymer/roster.html

```diff
@@ -57,6 +57,7 @@
       bottom: 122px;
       right: 30px;
       left: 30px;
+      min-height: 218px; /* TODO: keep downArrow and add button anchored to bottom when min-height kicks in. Ideally we'll get this for free with https://github.com/uProxy/uproxy/issues/2547 */
```

Comment:
So, on Chrome I see a weird line between the bubble and the arrow, like so:

![image](https://cloud.githubusercontent.com/assets/5142116/16886686/be8e3e62-4aa3-11e6-99e7-aa4628da4b39.png)